interface/src/app/home/home.tpl.html

```diff
@@ -1,32 +1,51 @@
 <div class="page-header">
   <h3>ADAGE: Home</h3>
+  The ADAGE web server allows users to analyze signatures from certain types
+  of unsupervised machine learning models. If you use the ADAGE server,
+  please cite:
+  <br>
+  Tan J, Huyck M, Hu D, Zelaya RA, Hogan DA, Greene CS.
+  <strong>
+    <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1905-4">
+      ADAGE signature analysis: differential expression analysis with
+      data-defined gene sets
+    </a>
+  </strong>
+  <em>BMC Bioinformatics. 2017.</em>
```

Comment:
Super-small thing - I think the publication year, 2017 shouldn't be italicized, is this right?

Reply:
I agree

Reply:
You guys are definitely right. When I convert the text written by @cgreene from md format to html, I made this mistake.